modules/gdscript/gdscript_editor.cpp

```diff
@@ -161,6 +161,10 @@ bool GDScriptLanguage::validate(const String &p_script, int &r_line_error, int &
 
 				funcs[cl->subclasses[i]->functions[j]->line] = String(cl->subclasses[i]->name) + "." + String(cl->subclasses[i]->functions[j]->name);
 			}
+			for (int j = 0; j < cl->subclasses[i]->static_functions.size(); ++j) {
```

Comment:
I used `++j` instead of `j++` because it uses correct semantics and expresses the wanted behaviour.

Reply:
Both have exactly the same result when used as a standalone statement. Actually, both carry over the same intent to the average programmer (increment the variable).
The main difference is that `x++` is used almost everywhere in the code, while `++x` is used only as an expression when its exact semantics are needed.

So, I think it would be better to change it back to `j++`, just in order to keep things consistent. If you feel like writing `++i` in loops is inherently better, feel free to open a discussion issue about it.

(apologies for the long 2¢)

Reply:
I read somewhere that certain c ++ compilers optimize ++i better than i++. Surely Microsoft compilers opt it the same. But maybe it's not a bad practice.

Reply:
I agree with @bojidar-bg, most of the codebase uses `x++` so there's no reason to do it differently here.

If you have strong feelings about `++x` being a super statement, we can discuss it and define a policy to use it or not, but until then, consistency is the priority.

Reply:
I do believe that ++i is inherently better but that's a discussion for another time.

Changing it.